src/pal_platform.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+#ifndef PAL_PLATFORM_H
+#define PAL_PLATFORM_H
+
+#include "pal/pal_device.h"
+#include "pal/pal_utils.h"
+#include "platform.h"
+#include "runtime.h"
+
+#include <atomic>
+#include <functional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <pal.h>
+#include <palLib.h>
+#include <palPipeline.h>
+#include <palPlatform.h>
+#include <palQueue.h>
+
+#ifdef AnyDSL_runtime_HAS_LLVM_SUPPORT
+#include <llvm/IR/DataLayout.h>
+#include <llvm/IR/LLVMContext.h>
+#include <llvm/Linker/Linker.h>
+#include <llvm/Pass.h>
+#include <llvm/Passes/OptimizationLevel.h>
+#include <llvm/Support/SourceMgr.h>
+#include <llvm/Target/TargetMachine.h>
+#endif
+
+namespace llvm_utils {
```

Comment:
This header is exposed to the rest of the runtime, but the `llvm_utils` namespace is only used inside `pal_platform.cpp`

Reply:
Good point. I'll remove it from the header.